gpu-computing.markdown

```diff
@@ -0,0 +1,110 @@
+---
+layout: page
+title: GPU Computing
+permalink: /gpu-computing/
+nav_order: 10
+---
+# GPU Computing
+
+
+## 1. Logging onto the cluster
+
+Nice intro `https://www.alessandravalcarcel.com/blog/2019-04-23-ssh/`
+
+```sh
+ssh -X <pennkey>@takim.pmacs.upenn.edu
+```
+
+```sh
+ssh -X <pennkey>@takim2.pmacs.upenn.edu
+```
+
+```sh
+ssh -X <pennkey>@takim2
+```
+
+
+## 2. Interactive Session Basics
+
+If you intend to use an interactive session, consider using screen so that you don't lose your work. You can open a screen by
+
+```sh
+screen -S <Screen-name>
+```
+
+```sh
+bsub -Is -q lpcgpu -gpu "num=1" -n 1 "bash"
+```
+
+Make sure you request gpu. "num=1" requests the number of GPU whereas "-n 1" requests the number of CPU.
+
+Next, load torch and tensorflow to activate CUDA
+
+```sh
+module load torch
+module load tensorflow/2.3-GPU
+```
+
+To check whether your CUDA is running, run this:
+
+```sh
+python
+```
+
+```py
+import torch
+torch.cuda.is_available()
+torch.cuda.deevice_count()
+torch.cuda.current_device()
+torch.cuda.device(0)
+torch.cuda.geet_device_name(0)
+```
+
+## 3. Batch Jobs Sessions
+
+## 4. Commands and References
+
+
+
+If that does not work then you need to install homebrew
```

Comment:
you probably want to delete the rest of the lines and add "Work in progress..." to the rest of the sections.